shared/featureconfig/config.go

```diff
@@ -180,11 +180,6 @@ func ConfigureBeaconChain(ctx *cli.Context) {
 		log.Warn("Disabled dynamic attestation committee subnets")
 		cfg.DisableDynamicCommitteeSubnets = true
 	}
-	cfg.EnableSSZCache = true
```

Comment:
Don't you have to remove the use of this value? Otherwise this PR is just disabling ssz cache.

Reply:
We could. But a lot of the tests will fail because they don't run well with state ssz cache. We'll need to investigate individual tests (~10 of them) and update state pkg to provide setters to clear the ssz cache for these tests to pass.

I was thinking to do that in subsequent PR. Do you prefer to do it in this PR?

Reply:
I would not accept this PR as is. This PR disables SSZ cache with no way to turn it on.

We either need to remove usage of this field, or leave line 183 untouched so that the default of ssz cache enabled is still true.

Reply:
Big face palm moment 🤦  
Line 183 should stay untouched. Thank you